entities/Session.js

```diff
@@ -0,0 +1,69 @@
+module.exports =  function(_, pg, db){
+	
+	function makeid(){
+        var text = "";
+        var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+        for( var i=0; i < 12; i++ )
+            text += possible.charAt(Math.floor(Math.random() * possible.length));
+        return text;
+    }
+
+    function createSession(data, callback){
```

Comment:
yep, and will spent more, but we will not write own session storage as same as we will not write own framework or own FB api wrapper because using an existed software will save ours and next another developers time and money